install.sh

```diff
@@ -0,0 +1,279 @@
+else
+  sudo wget -q -O /etc/profile.d/clearpath-ros-environment.sh \
+    https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/clearpath-ros-environment.sh
+  # Check if was added
+  if [ ! -e /etc/profile.d/clearpath-ros-environment.sh ]; then
+    echo -e "\e[31mError: CPR ROS environment exist, exiting\e[0m"
+    exit 0
+  fi
+fi
+
+if [ -e /etc/ros/setup.bash ]; then
+  echo -e "\e[33mWarn: CPR ROS robot environment exist, skipping\e[0m"
+else
+  sudo wget -q -O /etc/ros/setup.bash \
+    https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/setup.bash
+  sudo sed -i "s/UNKNOWN_ROS_DISTRO/${ros_version}/g" /etc/ros/setup.bash
+  # Check if was added
+  if [ ! -e /etc/ros/setup.bash ]; then
+    echo -e "\e[31mError: CPR ROS robot environment exist, exiting\e[0m"
+    exit 0
+  fi
+fi
+
+echo "source /opt/ros/${ros_version}/setup.bash" >> $HOME/.bashrc
+echo -e "\e[32mDone: Configuring Robot environment\e[0m"
+echo ""
+
+echo -e "\e[94mConfiguring rosdep\e[0m"
+if [ -e /etc/ros/rosdep/sources.list.d/20-default.list ]; then
+  echo -e "\e[33mWarn: rosdep was initalized, skipping\e[0m"
+else
+  sudo rosdep -q init
+  if [ ! -e /etc/ros/rosdep/sources.list.d/20-default.list ]; then
+    echo -e "\e[31mError: rosdep failed to initalize, exiting\e[0m"
+    exit 0
+  fi
+fi
+
+if [ -e /etc/ros/rosdep/sources.list.d/50-clearpath.list ]; then
+  echo -e "\e[33mWarn: CPR rosdeps exist, skipping\e[0m"
+else
+  sudo wget -q https://raw.githubusercontent.com/clearpathrobotics/public-rosdistro/master/rosdep/50-clearpath.list -O \
+    /etc/ros/rosdep/sources.list.d/50-clearpath.list
+  # Check if was added
+  if [ ! -e /etc/ros/rosdep/sources.list.d/50-clearpath.list ]; then
+    echo -e "\e[31mError: CPR rosdeps, exiting\e[0m"
+    exit 0
+  fi
+fi
+rosdep -q update
+echo -e "\e[32mDone: Configuring rosdep\e[0m"
+echo ""
+
+echo -e "\e[94mConfiguring udev rules\e[0m"
+sudo wget -q -O /etc/udev/rules.d/10-microstrain.rules \
+  https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/udev/10-microstrain.rules
+sudo wget -q -O /etc/udev/rules.d/41-clearpath.rules \
+  https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/udev/41-clearpath.rules
+sudo wget -q -O /etc/udev/rules.d/41-hokuyo.rules \
+  https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/udev/41-hokuyo.rules
+sudo wget -q -O /etc/udev/rules.d/41-gamepad.rules \
+  https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/udev/41-gamepad.rules
+sudo wget -q -O /etc/udev/rules.d/52-ftdi.rules \
+  https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/udev/52-ftdi.rules
+sudo wget -q -O /etc/udev/rules.d/60-startech.rules \
+  https://raw.githubusercontent.com/clearpathrobotics/ros_computer_setup/main/files/udev/60-startech.rules
+echo -e "\e[32mDone: Configuring udev rules\e[0m"
+echo ""
```

Comment:
I like the idea we'd had about moving all of these files into a udev-rules deb package.  Probably not worth doing for the initial release, but I think we should move forward with that plan eventually.

Reply:
Acked